admin/pre-reg/views.py

```diff
@@ -0,0 +1,171 @@
+import functools
+
+from django.contrib.auth.decorators import login_required  # , user_passes_test
+from django.views.decorators.csrf import csrf_exempt
+from django.contrib.auth.models import User
+from django.shortcuts import render
+from django.http import JsonResponse
+
+import json
+import httplib as http
+
+from modularodm import Q
+
+import utils
+
+from framework.auth.core import User as OsfUser
+from framework.mongo.utils import get_or_http_error
+from framework.exceptions import HTTPError, PermissionsError
+from website.project.model import MetaSchema, DraftRegistration
+from website.exceptions import NodeStateError
+
+get_draft_or_error = functools.partial(get_or_http_error, DraftRegistration)
+
+def load_osf_user(django_user):
```

Comment:
Doesn't need to be a database field but if you added:
```python
@property
def osf_user_instance():
    if not self.osf_user:
        raise RuntimeError()
    else:
        return OsfUser.load(self.osf_user.osf_id)
```

Then you could do:
```python
my_user_instance = MyUser.objects.all()[0]
osf_user_instance = my_user_instance.osf_user_instance
```

Django prefers fat-models over helper methods.